src/main/java/cc/polyfrost/oneconfig/config/Config.java

```diff
@@ -273,6 +287,131 @@ protected final void generateOptionList(Object instance, Class<?> targetClass, O
         logger.trace("Finished generating option list for {} (targetting={})", mod.name, targetClass.getName());
     }
 
+    /**
+     * Adds a category to the option list, for internal use only
```

Comment:
if this is for internal use only, mark it as such (i.e. add `internal` modifier and/or `@ApiStatus.Internal`